integration/plugin/authz/authz_plugin_test.go

```diff
@@ -176,15 +187,17 @@ func TestAuthZPluginAPIDenyResponse(t *testing.T) {
 	daemonURL, err := url.Parse(d.Sock())
 	assert.NilError(t, err)
 
-	conn, err := net.DialTimeout(daemonURL.Scheme, daemonURL.Path, time.Second*10)
+	client, err := socketHTTPClient(daemonURL)
```

Comment:
the `client` var is shadowing the `client` import; perhaps

```suggestion
	socketClient, err := socketHTTPClient(daemonURL)
```